### PR TITLE
Auth Pages on frontend

### PR DESCRIPTION
**Pull request by PaytonOAnderson**
Added framework for the Auth Pages on the frontend and changed login to route / and calendar routes to /calendar/.

### CODE REVIEW

File: packages/react-frontend/src/authPages/signupForm.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    }
+
+    const submitForm = async () => {
+        if (!user.username || !user.password) {
+            setErrorMessage("Please enter both username and password");
+            return;
+        }
+
+        try {
+            // Your fetch logic here...
```
Comment (Proefey) [inline packages/react-frontend/src/authPages/signupForm.js]: Remove This Comment

Comment (Proefey) [reply packages/react-frontend/src/authPages/signupForm.js]: Good Job On Organizing and Compartmentalizing The Code

**Review by lucawperri — APPROVED** [review]
Looks good Payton. The changes to MyApp and Calendar look good, and the auth page looks good!

**Comment by NipunDas** [comment]
Looks good to me. I think the new routes make more sense to a user. Nice work!

**Comment by Muhammad8493** [comment]
The new pages added look good and fit well with the rest of the application. Also, nice work on improving the routes in the app.

**Review by Muhammad8493 — APPROVED** [review]

**Comment by NipunDas** [comment]
Closing this PR since changes were merged into backend branch

**Pull request closed without merging**